docs/operations.rst

```diff
@@ -41,7 +41,7 @@ Healthchecks
 Logging
 =======
 
-By default KSQL server logs are written to ``/tmp/ksql-logs/``. 
+By default KSQL server logs are written to ``logs`` directory in KSQL folder.
```

Comment:
This isn't strictly accurate: it defers by distribution and mode of starting. I would rephrase this to:

> For local builds of ksql, the server logs are written to the ``logs`` directory in the KSQL folder.